cmd/promtool/main.go

```diff
@@ -20,8 +20,12 @@ import (
 	"path/filepath"
 	"strings"
 
+	yaml "github.com/ghodss/yaml"
```

Comment:
It's vendored already, it's entirely local to the package and gives us an immediate benefit. Any config/ refactoring is far away. I see no reason to not do this.